echopype/convert/parse_base.py

```diff
@@ -337,11 +386,23 @@ def _append_channel_ping_data(self, datagram, rx=True):
             the newly read sample datagram
         rx : bool
             whether this is receive ping data
+        zar_vars : bool
+            whether one should account for zar vars
```

Comment:
What's "zar vars"? Do you mean `zarr vars` as in zarr variables? If so, maybe `zarr_vars` might be more clear.

Reply:
Wow, that is a strange typo. Yes, I should change `zar_vars` to `zarr_vars` and correct any typos here.